ftw/http.py

```diff
@@ -365,7 +379,7 @@ def build_request(self):
                         'data': unicode(self.request_object.data),
                         'function': 'http.HttpResponse.build_request'
                     })                
-            request = string.replace(request, '#data#', data + self.CRLF)
+            request = string.replace(request, '#data#', data)
```

Comment:
The bottom fix is to make it more in compliance with HTTP. since users can themselves include \r\n when magic is turned off.